themes/cardinal_service/lib/scss/favorite/_dashboard.scss

```diff
@@ -0,0 +1,196 @@
+@charset 'UTF-8';
+
+.su-masthead {
+  @include grid-media-min('lg') {
+    .su-lockup {
+      flex: 0 0 50%;
+      max-width: 50%;
+    }
+
+    .su-site-search {
+      flex: 0 0 25%;
+      max-width: 25%;
+    }
+  }
+}
+
+.role--anonymous {
```

Comment:
Is it needed to change the styles for anonymous users?

Reply:
Not at all! I shall redo the specificity for this styling 👀